sabnzbd/utils/file_extension.py

```diff
@@ -255,14 +256,20 @@
 
 # Combine to one tuple, with unique entries:
 ALL_EXT = tuple(set(POPULAR_EXT + DOWNLOAD_EXT))
-# Prepend a dot to each extension, because we work with a leading dot in extensions
+# ... and prepend a dot to each extension, because we work with a leading dot in extensions
 ALL_EXT = tuple(["." + i for i in ALL_EXT])
 
 
+def all_extensions() -> Tuple[List[str]]:
```

Comment:
No, that's not what I said ;)
`-> Tuple[str, ...]`
See https://stackoverflow.com/a/54747327/5235502

Reply:
Your PyCharm must have also indicated this, it will highlight the `return` in slight yellow highlight to indicate it doesn't match what you specified.

Reply:
Oh, "-> Tuple[str, ...]" litterally? OK.